### PR TITLE
misc(docs): Deployment - build by Heroku

### PR DESCRIPTION
**Pull request by ovr**
Hello!

Thanks

### CODE REVIEW

**Review by igorlukanin — APPROVED** [review]
LGTM! Also, I like that you've added a link to Heroku's docs.

**Comment by igorlukanin** [comment]
@hassankhan Please have a look 🙏

File: docs/Deployment/Deployment-Guide.md [hunk 1]
```diff
@@ -364,3 +364,34 @@ $ heroku config:set REDIS_URL=<YOUR-REDIS-URL> -a cubejs-heroku-demo
...
+```
+
+Commit the file to your repo:
+
+```sh
+$ git add heroku.yml
+$ git commit -m "Add heroku.yml"
+```
+
+Set the stack of your app to container:
```
Comment (hassankhan) [inline docs/Deployment/Deployment-Guide.md:385]:
```diff
-container
+`container`
```

**Review by hassankhan — CHANGES_REQUESTED** [review]
Just the one nitpick, nice job @ovr 🎉

File: docs/Deployment/Deployment-Guide.md (hunk 1 above)
Comment (ovr) [reply docs/Deployment/Deployment-Guide.md]: 👍

**Pull request merged**